shenyu-common/src/test/java/org/apache/shenyu/common/config/ShenyuConfigTest.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.common.config;
+
+import org.junit.Test;
+import org.springframework.util.Assert;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Test cases for ShenyuConfig.
+ */
+public class ShenyuConfigTest {
+    private final ShenyuConfig config = new ShenyuConfig();
+
+    /**
+     * test of shenyuConfig.
+     *
+     * @return shenyuConfig
+     */
+    public ShenyuConfig getConfig() {
+        return this.config;
+    }
+
+    @Test
+    public void testShenyuConfig() {
+        ShenyuConfig.CrossFilterConfig cross = config.getCross();
+        ShenyuConfig.SwitchConfig switchConfig = config.getSwitchConfig();
+        ShenyuConfig.ExcludePath exclude = config.getExclude();
+        ShenyuConfig.FallbackPath fallback = config.getFallback();
+        ShenyuConfig.FileConfig file = config.getFile();
+        ShenyuConfig.InstanceConfig instance = config.getInstance();
+        ShenyuConfig.ExtPlugin extPlugin = config.getExtPlugin();
+        ShenyuConfig.Local local = config.getLocal();
+        ShenyuConfig.RibbonConfig ribbon = config.getRibbon();
+        ShenyuConfig.MetricsConfig metrics = config.getMetrics();
+        ShenyuConfig.Scheduler scheduler = config.getScheduler();
+        ShenyuConfig.SharedPool sharedPool = config.getSharedPool();
+        ShenyuConfig.WebsocketConfig websocket = config.getWebsocket();
+        ShenyuConfig.UpstreamCheck upstreamCheck = config.getUpstreamCheck();
+
+        assertNotNullValue(cross, switchConfig, exclude, fallback, file, instance,
+                extPlugin, local, ribbon, metrics, scheduler, sharedPool, websocket, upstreamCheck);
+    }
+
+    private void assertNotNullValue(final Object... objects) {
+        Assert.noNullElements(objects, "val not allow null");
```

Comment:
Does `java.lang.String` need to be considered here ? Such as `""`. By the way, there may be some negative numbers(such as `-1` for `int` or `long`).

Reply:
ok, I'll fix it

Reply:
Looking forward to your contribution :)